tests/system/Helpers/FormHelperTest.php

```diff
@@ -195,13 +182,7 @@ public function testFormOpenWithHidden()
 
     public function testFormOpenMultipart()
     {
-        $config            = new App();
-        $config->baseURL   = '';
-        $config->indexPage = 'index.php';
-        $request           = Services::request($config);
-        $request->uri      = new URI('http://example.com/');
-
-        Services::injectMock('request', $request);
+        $this->setRequest();
 
         $before = (new Filters())->globals['before'];
         if (in_array('csrf', $before, true) || array_key_exists('csrf', $before)) {
```

Comment:
Everything up to here seems to be about forms; did you intend for this to be part of the PR?

Reply:
Yes, my intention is just removing the code that set `IncomingRequest::$uri` value.
That is to remove `$request->uri = new URI('http://example.com/')`.
The property will be protected.